src/vcCamera.cpp

```diff
@@ -498,7 +500,7 @@ void vcCamera_HandleSceneInput(vcState *pProgramState, udDouble3 oscMove, udFloa
   }
 
   // Double Clicking left mouse
-  if (isBtnDoubleClicked[0] && (pProgramState->pickingSuccess))
+  if (isBtnDoubleClicked[0] && (pProgramState->udModelPickedIndex != -1))
```

Comment:
`pickingSuccess` should be false on the skybox so I don't think this fixes it- it will just break on non-UDS models

Reply:
Fixed pickingSuccess flag.